juriscraper/pacer/attachment_page.py

```diff
@@ -121,15 +151,98 @@ def _get_document_number(self):
                 continue
         return None
 
+    def _decrement_attachment_index(self, row):
+        """Return if we need to decrement the attachment index.
+
+        We need to do this for all bankruptcy courts and old attachment pages
+        in some district courts.
```

Comment:
I'm glad you put this comment because I didn't catch that this was an issue for all bankruptcy courts. But I'm confused because we don't have any bankruptcy attachment page tests that you're updating. Does that mean we didn't have tests we should or that bankruptcy was working OK before?

Reply:
I just moved the check for bankruptcy courts into this function, it already existed previously.

Reply:
Whew.